src/test/java/com/test/cucumber/RunCukes.java

```diff
@@ -0,0 +1,29 @@
+package com.test.cucumber;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+import org.testng.annotations.DataProvider;
+
+@CucumberOptions(
+        plugin = {
+                "com.cucumber.listener.CucumberScenarioReporterListener",
+                "com.cucumber.listener.CucumberScenarioListener",
+                "html:target/results.html",
+                "message:target/results.ndjson",
+                "timeline:target/timline"
+        })
+public class RunCukes extends AbstractTestNGCucumberTests {
+    public RunCukes() {
+        System.out.printf("ThreadID: %d: RunCucumberTest", Thread.currentThread().getId());
+    }
+
+    @Override
+    @DataProvider(parallel = true)
```

Comment:
Can this be part of the listener somehow by default rather than users specifying it?

Reply:
Need to see if that can be done. If yes, ofcourse.